CHANGELOG.md

```
@@ -1,3 +1,46 @@
+<a name="v0.3.2"></a>
+# [v0.3.2](https://github.com/travisghansen/hass-opnsense/releases/tag/v0.3.2) - 28 Sep 2024
+
+<!-- Release notes generated using configuration in .github/release.yml at main -->
+
+## What's Changed
+### 💥 Breaking Change 💥
```

Markdownlint

Expected: 1; Actual: 0; Above
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)